stubs/api.php

```diff
@@ -39,6 +39,18 @@
     'namespace' => null,
     'by-resource' => true,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Model Namespace
+    |--------------------------------------------------------------------------
+    |
+    | Here you can decide where your api models live.
+    |
+    | By default Models live on the root of the application like App\Post hence the model-namespace: 'App'
+    | but you can set it any other place like App\Models\Post hence the model-namespace: 'App\Models'
+    */
+    'model-namespace' => 'App',
```

Comment:
Default here should be `null` and the comments above should make clear the `null` will ensure that it defaults to the Laravel application's namespace.

Reply:
done